BoxBrowseSDK/BoxBrowseSDK/Categories/UIImage+BOXBrowseSDKAdditions.m

```diff
@@ -66,7 +65,9 @@ + (UIImage *)box_smallIconForItem:(BOXItem *)item
         NSString *fileExtension = [item.name box_pathExtensionAccountingForZippedPackages].lowercaseString;
         NSString *iconName = [self iconNameForFileExtension:fileExtension];
 
-        icon = [UIImage box_iconWithName:[@"small_" stringByAppendingString:iconName]];
+        if (iconName) {
```

Comment:
Regardless of safeguarding against nil, which we can keep, should we make sure there is a default value? like the 'generic' icon - is there not 'small_generic'?